ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -332,6 +334,11 @@ def waypoints_cb(self, lane_msg):
                           format(len(self.waypoints), self.waypoints[cntr-1].
                                  ptr_id, self.waypoints[cntr-1].get_s()))
             self.max_s = self.waypoints[cntr-1].get_s()
+            if self.max_s < 1000.0:
+                rospy.logwarn("Not in Track 1")
+                self.in_sim = False
+                self.initial_accel = self.initial_accel * 0.5
+                self.max_desired_jerk = self.max_desired_jerk * 0.66
```

Comment:
Instead of judging based on the length of waypoints, we could split the .launch file and move these parameters into there so they can be tuned separately for site vs sim launch, like TL detector and DBW nodes.